docs/user-guide/admin-cli.md

```diff
@@ -87,45 +87,106 @@ optional arguments:
 
 ### Create
 A vmgroup named "_DEFAULT" will be created automatically post install.
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup ls
+Uuid                                  Name      Description                Default_datastore  VM_list
+------------------------------------  --------  -------------------------  -----------------  -------
+11111111-1111-1111-1111-111111111111  _DEFAULT  This is a default vmgroup  __VM_DS
+```
+
+The "Default_datastore" field is set to "__VM_DS" for "_DEFAULT" vmgroup. Any volume create from VM which belongs to "_DEFAULT" vmgroup will be created on the datastore where VM resides.
+
+Two full access privileges will be created automatically post install.
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access ls --name=_DEFAULT
+Datastore  Allow_create  Max_volume_size  Total_size
+---------  ------------  ---------------  ----------
+__ALL_DS   True          Unset            Unset
+__VM_DS    True          Unset            Unset
+```
 
 Creates a new named vmgroup and optionally assigns VMs. Valid vmgroup name is only allowed to be "[a-zA-Z0-9_][a-zA-Z0-9_.-]*"
 
+"Default_datastore" is a required parameter when creating a new named vmgroup. Valid value for this field includes the name of
```

Comment:
Drop "when creating a new named vmgroup". You are in "vmgroup create" :-).
Also drop "for this field". Better to say "The value is either a valid datastore name, or special string "_VM_DS".